### PR TITLE
Adopt client-side verification key transform in latest zeth

### PR DESCRIPTION
**Pull request by dtebbs**
Also depends on PR: clearmatics/zeth#293

### CODE REVIEW

File: client/zecale/cli/__main__.py [hunk 1]
```diff
@@ -39,7 +39,13 @@ def __call__(self, *args: Any, **kwargs: Any) -> Any:
 @option(
     "--aggregator-server", "-a",
     default=AGGREGATOR_SERVER_ENDPOINT_DEFAULT,
-    help=f"Prover server endpoint (default={AGGREGATOR_SERVER_ENDPOINT_DEFAULT})")
+    help="Aggregation server endpoint "
```
Comment (AntoineRondelet) [inline client/zecale/cli/__main__.py:42]: `Aggregation server` -> `Aggregator server` ?

File: libzecale/tests/serialization/proto_utils_test.cpp [hunk 2]
```diff
@@ -43,87 +73,15 @@ template<typename ppT> void test_parse_nested_transaction_pghr13()
...
+    test_parse_nested_transaction<
+        ppT,
+        libzeth::pghr13_snark<ppT>,
+        libzeth::pghr13_api_handler<ppT>>(mock_extended_proof);
 }
 
 template<typename ppT> void test_parse_nested_transaction_groth16()
 {
-    // 1. Format arbitary data that will be parsed afterwards
+    // Format arbitary data that will be parsed afterwards
```
Comment (AntoineRondelet) [inline libzecale/tests/serialization/proto_utils_test.cpp:84]: `arbitary` -> `arbitrary`

**Comment by AntoineRondelet** [comment]
Implements similar approach as https://github.com/clearmatics/zeth/issues/292 to fetch config from `aggregator_server`

File: libzecale/tests/serialization/proto_utils_test.cpp (hunk 2 above)
Comment (dtebbs) [reply libzecale/tests/serialization/proto_utils_test.cpp]: fixed.  thanks.

**Review by AntoineRondelet — APPROVED** [review]
LGTM

**Pull request merged**